spec/controllers/organizations_controller_spec.rb

```diff
@@ -0,0 +1,26 @@
+RSpec.describe OrganizationsController, type: :controller do
+  let(:default_params) do
+    { organization_id: @organization.to_param }
+  end
+
+  context "While signed in as org admin" do
+    before do
+      sign_in(@organization_admin)
+    end
+    
+    describe "POST #update" do
+      context "When all distribution types are disabled" do
+        subject { put :update, params: default_params.merge( 
+          enable_child_based_requests: false, 
+          enable_individual_requests: false, 
+          enable_quantity_based_requests: false )
+        }
+
+        it "Returns flash error indicating at least one distribuion type must be selected" do
+          expect(flash[:error]).to eq("enable_requests: You must allow at least one request type (child-based, individual, or quantity-based)")
```

Comment:
You might find more success using a `match` matcher using regex. I noticed the flash contains some HTML in there.

Note - I don't believe we ought to put HTML in the flash message but if we were I think thats how I would handle the testing.

Reply:
After doing some reading, I saw that it is not recommended to use controller tests but instead to use request tests so I'm going to delete this file. The change I mentioned in my other comment will also make testing at this layer unnecessary.